extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -53,7 +56,72 @@ export class AzdataTool extends ToolBase {
 		return true;
 	}
 
-	public install(): Promise<void> {
-		return Promise.reject('not implemented');
+	protected get installationPath(): Promise<string | null> {
```

Comment:
Promise return type should be async https://palantir.github.io/tslint/rules/promise-function-async/

Reply:
Actually I agree in general. There has been the sentiment of marking async only when await is in the body of methods that return a Promise, but I think that leads to harder to read and maintain code. 

However in this particular case the language does not allo to mark a 'getter' as async so I cannote do it unless I change installatinoPath to a regular getInstallationPath() method.